### PR TITLE
GEN-1213 | Display total amount on trial offer page

### PR DESCRIPTION
**Pull request by robinandeer**
### With extension
n

### Without extension

- Display `TotalAmount` on car trial offer page

- Tweak the layout of the page (spacing)

## Justify why they are needed

- Updates from design

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3283** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3283" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3283?utm_source=stack-comment).

File: apps/store/src/features/carDealership/TrialExtensionForm.tsx [hunk 1]
```diff
@@ -14,6 +17,7 @@ const SIGN_BUTTON = 'Sign insurance'
 const CONTINUE_WITHOUT_EXTENSION_BUTTON = 'Connect payment in the app'
 const INFO_CARD_CONTENT = 'Se allt om din prova på-försäkring i Hedvig-appen.'
 const UNDO_REMOVE_BUTTON = 'Undo removal'
+const COST_EXPLANATION = 'discounted price until {}'
```
Comment (alebedev) [inline apps/store/src/features/carDealership/TrialExtensionForm.tsx:20]: Do you see those texts going into Lokalize or CMS?

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/features/carDealership/TrialExtensionForm.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/features/carDealership/TrialExtensionForm.tsx:20]: I think most of them are just translated text strings

But let's discuss with design how they see it

Not sure if we discussed if we want to support english...

**Pull request merged**